stories/top-bar.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { TopBar } from '../src/components/top-bar/top-bar';
+
+
+storiesOf('TopBar', module)
```

Comment:
So I do have a pending PR to update all this old syntax: #568 . I'd prefer not to have more stories before that is merged

Reply:
agree, i have remove the stories and will add when the pending  PRs get merged